docs/en/observability/metrics-app-fields.asciidoc

```diff
@@ -76,7 +76,7 @@ example: `MacBook-Elastic.local`
 
 IP of the host that records the event.
 +
-type: ip
+type: IP
```

Comment:
These should be formatted in backticks, not capitalized because `ip` is an Elasticsearch type: https://www.elastic.co/guide/en/elasticsearch/reference/current/ip.html